code/dao/base_dao.py

```diff
@@ -0,0 +1,62 @@
+from typing import TypedDict, Optional
+
+from config.db import DBManager
+
+
+class ListParams(TypedDict, total=False):
+    sort: Optional[str]
+    query: Optional[str]
+    page: Optional[int]
+    limit: Optional[int]
+
+
+class BaseDAO:
+
+    def __init__(self, table: str, db_url: str):
+        self._db = DBManager(db_url)
+        self._cur = self._db.connection.cursor()
+        self._table = table
+
+    def get_all(self, params: ListParams = None):
+        if params is None:
+            params = {}
+
+        sql = "SELECT * FROM %s"
```

Comment:
Uf, más detalles de implementación... no tento muy claro por qué está esto aquí (una vez más, porque no tiene nada que ver con el PR)

Reply:
removed (for now) all these stuff...